common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -1202,3 +1202,107 @@ def test_invalidate_certificates_a11y(self):
             '.certificates-wrapper'
         ])
         self.certificates_section.a11y_audit.check_for_accessibility_errors()
+
+
+@attr(shard=10)
+class EcommerceTest(BaseInstructorDashboardTest):
+    """
+    Bok Choy tests for the "E-Commerce" tab.
+    """
+    def setUp(self):
+        super(EcommerceTest, self).setUp()
+
+    def setup_course(self, course_number):
+        """
+        Sets up the course
+        """
+        self.course_info['number'] = course_number
+        course_fixture = CourseFixture(
+            self.course_info["org"],
+            self.course_info["number"],
+            self.course_info["run"],
+            self.course_info["display_name"]
+        )
+        course_fixture.install()
+
+    def log_in_as_unique_user(self):
+        """
+        Log in as a valid lms user.
+        """
+        AutoAuthPage(
+            self.browser,
+            username="test_instructor",
+            email="test_instructor@example.com",
+            password="password",
+            course_id=self.course_id
+        ).visit()
+
+    def visit_ecommerce_section(self):
+        """
+        Log in to visit Instructor dashboard and click E-commerce tab
+        """
+        self.log_in_as_unique_user()
+        instructor_dashboard_page = self.visit_instructor_dashboard()
+        return instructor_dashboard_page.select_ecommerce_tab()
+
+    def add_course_mode(self, sku_value=None):
+        """
+        Add an honor mode to the course
+        """
+        ModeCreationPage(self.browser, self.course_id, min_price=10, sku=sku_value).visit()
```

Comment:
be consistent here. 
`ModeCreationPage(browser=self.browser, course_id=self.course_id, min_price=10, sku=sku_value).visit()`
why are not we using `mode_slug` here for honor ? if not any complication then please add too. 
I am also curious about `sku` field here. Is `ModeCreationPage` handling the `sku` ??